client/src/lib.rs

```diff
@@ -1402,6 +1413,40 @@ impl WorkflowClientTrait for Client {
             .into_inner())
     }
 
+    async fn update_workflow_execution(
+        &self,
+        workflow_id: String,
+        run_id: String,
+        name: String,
+        wait_policy: update::v1::WaitPolicy,
+        args: Option<Payloads>,
+    ) -> Result<UpdateWorkflowExecutionResponse> {
+        Ok(self
+            .wf_svc()
+            .update_workflow_execution(UpdateWorkflowExecutionRequest {
+                namespace: self.namespace.clone(),
+                workflow_execution: Some(WorkflowExecution {
+                    workflow_id,
+                    run_id,
+                }),
+                wait_policy: Some(wait_policy),
+                request: Some(update::v1::Request {
+                    meta: Some(update::v1::Meta {
+                        update_id: "".into(),
```

Comment:
Why are you not passing an update ID?

Reply:
I just wanted to keep the signature small in this trait. Rust client users can go raw grpc if they need to specify it.